components/credit-card/CardColorProblem.tsx

```diff
@@ -1,10 +1,195 @@
-import React, { useState } from 'react'
-import { CardColorProblemData } from '../../pages/api/credit-card/CardColorProblemData'
-import FinanceCard from './FinanceCard';
+import React from "react";
+import { CardColorProblemData } from "../../pages/api/credit-card/CardColorProblemData";
+import FinanceCard from "./FinanceCard";
 
-const CardColorProblem = () => {
+interface CardColorProps {
+  cardQ1: string;
```

Comment:
I know this feels super bloated, a nicer way to achieve this would have been to use Redux. Hoping for you to get familiar with Redux soon.